spec/array_inject_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'array_inject'
+
+describe SubclassArray do 
+	it 'has a working inject method' do 
+		expect(subject.injecting([5,6,7,8,9,10],0)).to eq 45
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing after comma.<br>Tab detected.